proto/ibc/applications/interchain_accounts/host/v1/host.proto

```diff
@@ -10,6 +10,8 @@ import "gogoproto/gogo.proto";
 // Params defines the set of on-chain interchain accounts parameters.
 // The following parameters may be used to disable the host submodule.
 message Params {
-  // host_enabled enables or disables the host submodule. 
+  // host_enabled enables or disables the host submodule.
   bool host_enabled = 1 [(gogoproto.moretags) = "yaml:\"host_enabled\""];
+  // allow_messages defines a list of sdk message typeURLs allowed to be executed on a host chain.
+  repeated string allow_messages = 2 [(gogoproto.moretags) = "yaml:\"allow_messages\""];
```

Comment:
I named this `allow_messages` as I thought `allow_list` was maybe a little ambiguous. 
Open to renaming